includes/admin/views/tools/general.php

```diff
@@ -10,20 +9,40 @@
 
         <form method="post" class="erp-tools-form" action="<?php echo esc_url( admin_url( 'admin.php?page=erp-tools' ) ); ?>">
             <?php
-            $menus          = get_option( '_erp_admin_menu', [] );
+            global $menu;
+            $menus = [];
+
+            foreach ( $menu as $single_menu ) {
+                if ( ! empty ( $single_menu[0] ) && ! empty ( $single_menu[2] ) && $single_menu[2] !== 'erp' ) { // Without ERP
+                    $single_menu[0] = explode( '<', $single_menu[0] )[0]; // Remove if any <span> inside title
+                    array_push( $menus, $single_menu );
+                }
+            }
+
+            $inactive_menu_options = get_option( '_erp_admin_menu', [] );
+            $inactive_menus        = [];
+
+            if ( is_array( $inactive_menu_options ) ) {
+                foreach( $inactive_menu_options as $inactive_menu ) {
```

Comment:
After `foreach` use a whitespace before the opening parentheses as coding standard.
And instead of checking `is_array( $inactive_menu_options )`, it might be done by casting the variable inside foreach loop like `foreach ( (array) $inactive_menu_options as $inactive_menu )` if there is no other side effect.